XCUITest/SearchSuggestionsTest.swift

```diff
@@ -67,7 +67,10 @@ class SearchSuggestionsPromptTest: BaseTestCase {
 
         // Ensure prompt disappears
         waitforNoExistence(element: app.otherElements["SearchSuggestionsPromptView"])
-
+        
+        // Adding a delay in case of slow network
+        sleep(4)
```

Comment:
Once the cause is timing on BB due to slow responses. We could try to do something like we have on firefox-iOS: [waitForExistence("element", timeout: xSeg)](https://github.com/mozilla-mobile/firefox-ios/blob/main/XCUITests/FindInPageTest.swift#L14) to do that we added to the BaseTestCase that method, [waitForExistence](https://github.com/mozilla-mobile/firefox-ios/blob/main/XCUITests/BaseTestCase.swift#L73), based on waitFor which is defined also on [Focus BaseTestCase](https://github.com/mozilla-mobile/focus-ios/blob/master/XCUITest/BaseTestCase.swift#L169). 
But you can do that later in a follow up or I can do it later.